tests/test_adjoint_jacobian.py

```diff
@@ -324,7 +324,7 @@ def test_gradient_gate_with_multiple_parameters(self, tol, dev_kokkos):
         grad_PS = fn(qml.execute(gtapes, dev_kokkos, gradient_fn=None))
 
         # gradient has the correct shape and every element is nonzero
-        assert len(grad_D) == 3
+        assert np.size(grad_D) == 3
```

Comment:
@mlxd @multiphaseCFD This assertion was failing, so I had to change it. This is because grad_D is two-dimensional, basically. Is that ok?

Reply:
The new return types updates should have fixed this and it should be 1D now https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/51/files

I'll take a look.